app/src/main/java/com/nutomic/syncthingandroid/service/EventProcessor.java

```diff
@@ -170,8 +181,23 @@ public void onEvent(Event event) {
             case "Ping":
                 // Ignored.
                 break;
-            default:
+            case "DeviceConnected":
+            case "FolderPaused":
+            case "FolderScanProgress":
+            case "FolderSummary":
+                // ToDo in a separate PR
+                // Call updateFolderStatus(RestApi api) to update the folder status UI
+            case "ItemStarted":
+            case "LocalIndexUpdated":
+            case "LoginAttempt":
+            case "RemoteIndexUpdated":
+            case "Starting":
+            case "StartupComplete":
+            case "StateChanged":
                 Log.v(TAG, "Unhandled event " + event.type);
```

Comment:
These are handled, and ignored, there is a whole explicit switch statement handling these, no point logging it.

Reply:
As I explained before, I like the log. It is just verbose level. ApiRequest is going wild on logging btw as it logs every second it does a query. We should something do about this limiting log spam.

Reply:
Or maybe we should call the message ignored event? Mostly devs care about verbose log and I haven't time to begin every coding session on a new topic with "oh my log points and indicators what st does when and why are missing and have to be inserted and later deleted". Plus, I can ask an experienced user to filter for eventprocessor to see if events were read, handled, ignored, unknown and the main plot of st usage was. Strategically this app needs a lot of bug fixes, enhancements that can easier be done with extensive logging. If the app reached a certain level of biggest bugs resolved for months, we can drop much logging or make it optional.

Reply:
So every time I had to look at someones provide logs from android, the log was full of lines telling me about events it didn't handle. I can read the code, I don't need the log to tell me that.

Reply:
User says: after my folder download completed syncthing crashed. Log is full of other things but event would have told us. If you want to filter something out, use excel grep and so on.

Reply:
What would a log line telling me RemoteIndexUpdated was dropped would have told me? The app does not use that, behavior of the app on wether it got logged or not does not change, so what value does it provide?

If I care about the events, I enable logging in synching, so I can see the payloads of the events not just the mere fact that it was emitted, god knows for what device what folder combination.

Sure, I can filter logs and what not, but it would be nice to have the log consumable out of the box eithout having to run it through grep.

Logs should be useful, not just verbose.

Reply:
So lets find an agreement in the middle and only log that details in a debug build. That would be enough for me having the logs when working on the code.
https://github.com/syncthing/syncthing-android/pull/1079/commits/6e5293d79e797703f434732852abb5487525addc